lib/internal/Magento/Framework/App/Cache/FlushCacheByTags.php

```diff
@@ -22,28 +29,26 @@ class FlushCacheByTags
     private $cacheList;
 
     /**
-     * @var StateInterface
+     * @var \Magento\Framework\App\Cache\StateInterface
      */
     private $cacheState;
 
     /**
-     * @var Tag\Resolver
+     * @var \Magento\Framework\App\Cache\Tag\Resolver
      */
     private $tagResolver;
 
     /**
-     * FlushCacheByTags constructor.
-     *
-     * @param Type\FrontendPool $cachePool
-     * @param StateInterface $cacheState
+     * @param \Magento\Framework\App\Cache\Type\FrontendPool $cachePool
+     * @param \Magento\Framework\App\Cache\StateInterface $cacheState
      * @param array $cacheList
-     * @param Tag\Resolver $tagResolver
+     * @param \Magento\Framework\App\Cache\Tag\Resolver $tagResolver
```

Comment:
FQCN are not needed here, please revert those changes, amend commit and force push.

Reply:
I've just updated the whole phpdoc with advisable settings. FQCN seems more readable here.
If you think it's not needed, please clarify where, (while modified files, constructor, methods...)

Reply:
FQCN must never be added, that's the rule. Please revert undesired additions and amend commit.